justspacesapp/urls.py

```diff
@@ -5,32 +5,41 @@
 refigure it to a more strictful RESTful approach.
 
 """
-from django.conf.urls import url
+from django.conf.urls import url, include
 from django.utils.translation import ugettext_lazy as _
+from django.contrib.auth.decorators import login_required, permission_required
 from justspacesapp.views import AgencyCreate, LocationCreate, StudyCreate, \
-                                SurveyList
+                                SurveyList, Signup
 import fobi.views
 
 urlpatterns = [
     url(r'^$',
-        SurveyList.as_view(),
+        login_required(SurveyList.as_view()),
```

Comment:
It seems like, eventually, we'll want the home view to be a friendly public-facing page, e.g., something we build in Part 2. Can we name this URL "login"? Then, we can avoid having to edit this later on.

Reply:
I'm intending this as a placeholder home page to use until we have the public-facing site views. I already have a url named login (`accounts/login`). I've changed the login view redirect to `survey-list` and added a comment to the home view. Does that address your concern here?

Reply:
yes!